anta/reporter/csv_reporter.py

```diff
@@ -58,8 +58,7 @@ def split_list_to_txt_list(cls, usr_list: list[str], delimiter: str = " - ") ->
 
     @classmethod
     def convert_to_list(cls, result: TestResult) -> list[str]:
-        """
-        Convert a TestResult into a list of string for creating file content.
+        """Convert a TestResult into a list of string for creating file content.
```

Comment:
I think we already talked about this but there's no way of enforcing this (not carriage return after the `"""`) in ruff right?

Reply:
it is already enforced in this PR cd pyproject D212 addition: https://docs.astral.sh/ruff/rules/multi-line-summary-first-line/